master/internal/resourcemanagers/kubernetes/pods.go

```diff
@@ -80,6 +80,15 @@ type pods struct {
 	configMapInterface typedV1.ConfigMapInterface
 }
 
+// PodsInfo contains information for pods.
+type PodsInfo struct {
+	NumAgents int
+	NumSlots  int
```

Comment:
I wonder if it'll look cleaner to call it `SlotsAvailable` to align it with the field name in the api structure.